nixos/doc/manual/from_md/release-notes/rl-2305.section.xml

```diff
@@ -214,6 +214,14 @@
           or configure your firewall.
         </para>
       </listitem>
+      <listitem>
+        <para>
```

Comment:
needs regeneration (I know this is super annoying)

Reply:
all good, I am both new to this, and happy to make it all look pretty